.cirrus.yml

```diff
@@ -0,0 +1,77 @@
+compute_engine_instance:
```

Comment:
What does this file do? Does it need documentation?

Reply:
This is a cirrus CI config. Cirrus CI provides FreeBSD runners for free. Documentation is here: https://cirrus-ci.org/guide/writing-tasks/